index.html

```diff
@@ -29,17 +29,17 @@
 
             <div class="menu-content">
               <ul>
-                <li><a href="#">Portfolio</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#">Contact</a></li>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
```

Comment:
- [ ] When the user clicks (or taps) any option of the mobile menu, the mobile menu is meant to disappear. I request that you hide the mobile menu after the user selecting an option.